exasol/slc/models/deploy_result.py

```diff
@@ -0,0 +1,10 @@
+from attr import dataclass
+
+from exasol.slc.models.language_definitions_builder import LanguageDefinitionsBuilder
+
+
+@dataclass
+class DeployResult:
+    release_path: str
+    upload_url: str
```

Comment:
Maybe we want to make this SaaS BucketFS compatible, maybe we actually return a BucketFS PathLike

Reply:
I can return both for now. The BucketFS Pathlike has some limitiations as we have seen.